unittests/fork_db_tests.cpp

```diff
@@ -167,6 +167,19 @@ BOOST_FIXTURE_TEST_CASE(add_remove_test, generate_fork_db_state) try {
    BOOST_TEST(branch[1] == bsp11b);
 } FC_LOG_AND_RETHROW();
 
+BOOST_FIXTURE_TEST_CASE(remove_block_num_test, generate_fork_db_state) try {
+   BOOST_TEST(fork_db.size() == 14);
+   fork_db.remove(13); // remove all >= 13
+   BOOST_TEST(fork_db.size() == 8);
```

Comment:
Why does `size` become `8`?

The following `for` loop is sufficient to show all blocks greater than or equal to 13 have been removed.

Reply:
Not sure I understand the question. Are you suggesting just remove the `BOOST_TEST(fork_db.size() == 8);` since the `for` loop verifies what was removed is correct?

Reply:
Sorry I confused you. Initially I was trying to figure out why it became `8`. Then I saw the `for` loop verifies they are all removed. You could add a comment saying why it is `8` now or just remove the size check.